src/core/friendly_errors/fes_core.js

```diff
@@ -47,12 +47,20 @@ import { translator } from '../internationalization';
 // p5.js blue, p5.js orange, auto dark green; fallback p5.js darkened magenta
 // See testColors below for all the color codes and names
 const typeColors = ['#2D7BB6', '#EE9900', '#4DB200', '#C83C00'];
+let misusedAtTopLevelCode = null;
+let defineMisusedAtTopLevelCode = null;
+
+// the threshold for the maximum allowed levenshtein distance
+// used in misspelling detection
+const EDIT_THRESH = 2;
```

Comment:
Maybe this is a levenshtein-specific term but I am thrown off by the word 'edit' here and on L179 `editDistance()`. I  associate 'edit' with writing not reading.

Reply:
Ah I see now that 'edit distance' is a [technical term you are referencing](https://en.wikipedia.org/wiki/Edit_distance). This is misleading without that knowledge; could potentially be `EDIT_DIST_THRESHOLD`.

Reply:
Fair enough. I will change this too